infrastructureModul/pom.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>by.VitaliPodelinski</groupId>
+    <artifactId>TravelAgency</artifactId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
+  <groupId>by.VitaliPodelinski</groupId>
+  <artifactId>infrastructureModul</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <name>infrastructureModul</name>
```

Comment:
not necessary to declare, just use the same name as you have declared in parent module